libs/indibase/inditelescope.cpp

```diff
@@ -3027,6 +3042,20 @@ bool Telescope::getSimulatePierSide() const
     return m_simulatePierSide;
 }
 
+const char * Telescope::getPierSideStr(TelescopePierSide ps)
+{
+    switch (ps)
+    {
+    case PIER_WEST:
+        return "PIER_WEST";
+    case PIER_EAST:
+        return "PIER_EAST";
+    case PIER_UNKNOWN:
```

Comment:
Would the warning go away if you add default along with PIER_UNKNOWN?

Reply:
Replaced PIER_UNKNOWN with default and use IUGetConfigSwitch.